devenv/dev-dashboards/panel-dashlist/dashlist.json

```
@@ -0,0 +1,86 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": {
+          "type": "grafana",
+          "uid": "-- Grafana --"
+        },
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "type": "dashboard"
+      }
+    ]
+  },
+  "editable": true,
+  "fiscalYearStartMonth": 0,
+  "graphTooltip": 0,
+  "id": null,
+  "links": [],
+  "liveNow": false,
+  "panels": [
+    {
+      "datasource": {
+        "type": "grafana-testdata-datasource",
+        "uid": "PD8C576611E62080A"
+      },
+      "gridPos": {
+        "h": 12,
+        "w": 12,
+        "x": 0,
+        "y": 0
+      },
+      "id": 2,
+      "options": {
+        "folderUID": "",
+        "includeVars": true,
+        "keepTime": false,
+        "maxItems": 10,
+        "query": "",
+        "showHeadings": true,
+        "showRecentlyViewed": true,
+        "showSearch": false,
+        "showStarred": false,
+        "tags": []
+      },
+      "pluginVersion": "10.3.0-pre",
+      "title": "Include time range and variables enabled",
+      "type": "dashlist"
+    }
+  ],
+  "refresh": "",
+  "schemaVersion": 39,
+  "tags": ["gdev", "panel-tests"],
+  "templating": {
+    "list": [
+      {
+        "current": {
+          "selected": true,
+          "text": "A",
+          "value": "B"
```

shouldn't these be the same?
```suggestion
          "text": "A",
          "value": "A"
```

---

@torkelo why would we want a different text and value in our dev dashboard? 🤔 given the options are A,B,C, or D, having `"text": "A", "value": "B"` feels very confusing 😅

---

nm, see you just fixed this 👍

---

@ashharrison90 was a type, but does not really matter here this part of the persisted model is not really used

---

but pushed an update so they are the same